dgen_os/python/input_data_functions.py

```diff
@@ -334,12 +369,12 @@ def melt_year(parameter_name):
     
     Parameters
     ----------
-    parameter name : 'string'
+    parameter_name : `string`
         Name of the parameter value in dataframe. 
     
     Returns
     -------
-    function : 'function'
+    function : `function`
```

Comment:
```suggestion
    function ::class: `pandas.DataFrame`
```
The internal function technically returns a DataFrame which is then returned by melt_year